applications/schema/schema/components.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Components",
+  "description": "components",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^[a-zA-Z][a-zA-Z0-9_]*[a-zA-Z0-9]$": {
+      "title": "Component Name",
+      "description": "The name of the component",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "component": {
+          "title": "Component",
+          "description": "The name of the component class",
+          "type": "string"
```

Comment:
do you think we should move away from the option to specify "package" and "class" separately? In the component description we support both kinds of registration, but it could be simpler to just require the "new" (1 year old) reduced registration syntax.

Reply:
To be specific and match the component SDK, I would change the description field to 
"The fully qualified class name used to register the component."

Reply:
For your first question, yes I don't see the advantage of keeping the old kind too, its and additional (and even a bit duplicated) line.